### DISCUSSION TITLE
Can you use model from civitai?

### DISCUSSION THREAD

**Discussion by dennypool** [azuritecoin/OnnxDiffusersUI#80] [General]
Can use model from civitai? hypernetwork model too?

**Comment by ClashSAN** [comment]
if the model is .ckpt, convert it to diffusers. then you need to convert to onnx.

**Comment by azuritecoin** [comment]
The CivitAI model currently fails when trying to convert to diffusers.

**Comment by rieve-nailo** [comment]
I was able to add a civitai model using this
[https://github.com/Amblyopius/Stable-Diffusion-ONNX-FP16](url)
